src/Administration/Resources/app/administration/src/module/sw-extension/component/sw-extension-card-base/sw-extension-card-base.scss

```
@@ -92,6 +92,15 @@
         &-version {
             font-weight: $font-weight-semi-bold;
         }
+
+        &-privilege-review {
+            font-size: 10px;
+            font-weight: $font-weight-semi-bold;
+            border-radius: $border-radius-pill;
+            background-color: var(--color-background-attention-default);
+            color: $color-black;
+            padding: 2px 4px;
```

We can use the new css variables here (we try to remove all scss variables)
```suggestion
            font-size: 0,625rem; // we have no variable for this, but rem should be okay
            font-weight: var(--font-weight-semi-bold);
            border-radius: 100%;
            background-color: var(--color-background-attention-default);
            color: var(--color-black);
            padding: 2px 4px;
```

---

nice thanks, implemented!